package tutorial.orcas.combat.server.event;

// EVENT_SCENE_MOVE_STEP
message EventSceneMoveStep {
  required int32 combat_id = 1;
  required int32 warrior_id = 2;
}
